generators/common/templates/sonar-project.properties.ejs

```diff
@@ -2,6 +2,10 @@ sonar.projectKey=<%= baseName %>
 sonar.projectName=<%= baseName %> generated by jhipster
 sonar.projectVersion=1.0
 
+# After changing password in SonarQube change this also here
+#sonar.login=admin
+#sonar.password=admin
```

Comment:
```suggestion
sonar.login=admin
sonar.password=admin
```

This is working out of the box, so I prefer to uncomment this.
If user logs in to web interface then user is forced to change password. After password change new password is needed to put also here.